gbpservice/nfp/orchestrator/modules/device_orchestrator.py

```diff
@@ -810,13 +812,18 @@ def _post_configure_device_graph(self, nfp_context):
     
     def device_up(self, event):
         nfp_context = event.data
-
+        nf_id = nfp_context['network_function']['id']
         # Get the results of PLUG_INTERFACES & PERFORM_HEALTH_CHECK events
         # results.
         results = event.graph.get_leaf_node_results(event)
 
         for result in results:
             if result.result.lower() != 'success':
+                if result.id == "PLUG_INTERFACES":
```

Comment:
Move this plug_interfaces_failed code to plug_interfaces() method in drivers/orchestration_driver.py